packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/index.tsx

```diff
@@ -29,6 +30,7 @@ class Banners extends React.PureComponent<Props> {
   componentDidMount() {
```

Comment:
One of the things talked in "jsync" is to refactor the components with old structure when we make any changes inside them

Reply:
Yeah, I understand, if it's ok I will start with the new structure in the next task

Reply:
you can leave this for now if you want. but your new component needs to be functional and hooks

Reply:
this is fixed!